tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NetActivitySdkTests.cs

```diff
@@ -0,0 +1,84 @@
+// <copyright file="NetActivitySdkTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    [UsesVerify]
+    public class NetActivitySdkTests : TracingIntegrationTest
+    {
+        public NetActivitySdkTests(ITestOutputHelper output)
+            : base("NetActivitySdk", output)
+        {
+            // Intentionally unset service name and version, which may be derived from OTEL SDK
+            SetServiceName(string.Empty);
+            SetServiceVersion(string.Empty);
```

Comment:
For this example we're no longer using the OTEL SDK to configure the service and version in code, so we should probably change this to look similar to our other test cases:

```suggestion
            SetServiceVersion("1.0.0");
```